emission/storage/timeseries/builtin_timeseries.py

```diff
@@ -261,6 +264,9 @@ def get_entry_at_ts(self, key, ts_key, ts):
         logging.debug("get_entry_at_ts result = %s" % retValue)
         return retValue
 
+    def getEntryList(self):
+        return self.entryList
+       
```

Comment:
again, this is wrong as well. you can't return **the** entry list because there is not just one.

Reply:
there are existing methods to get an entry list. the trip model uses existing methods. You should not have to make changes to the `BuiltinTimeseries` for this change.